file-formats/tabl/type/zif_aff_indx_v1.intf.abap

```diff
@@ -0,0 +1,42 @@
+INTERFACE zif_aff_indx_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Index Fields</p>
+    "! Index fields
+    BEGIN OF ty_indexfield,
+      "! <p class="shorttext">Field Name</p>
+      "! Field name
+      fieldname TYPE c LENGTH 30,
+      "! <p class="shorttext">Field Position</p>
+      "! Position
+      position  TYPE n LENGTH 4,
+    END OF ty_indexfield,
+    tt_index_fields TYPE SORTED TABLE OF ty_indexfield WITH UNIQUE KEY fieldname.
+
+  TYPES:
+    "! <p class="shorttext">ABAP DDIC Secondary Index Properties</p>
+    "! Object type INDX
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
For REPS & FUNC, we use the header `zif_aff_types_v1=>ty_header_only_description`, a header for sub objects which consists only of the field `description`. Should this be kept the same?

Reply:
The original language can be derived from the table. That is not necessary true for the abap_language_version. THere are still discussion going on, but currently we must assume that extension indexes (R3TR XINX) can have an abap_labguage_version independent from the table. 
So far we assume that the file format for LIMU INDX and R3TR XINX is the same. If this is still true this would mean that we leave the header field as it is, correct?